packages/mjml-core/src/createComponent.js

```diff
@@ -87,6 +87,13 @@ export class BodyComponent extends Component {
     return shorthandParser(mjAttribute, direction)
   }
 
+  getShorthandBorderValue(direction) {
+    const borderDirection = this.getAttribute(`border-${direction}`)
+    const border = this.getAttribute('border')
```

Comment:
how does it works when you have something like `border: 1px solid red` ? Will it parse it correctly?